test_stbt_rig.py

```diff
@@ -225,3 +225,23 @@ def test_run_tests_interactive(capsys, test_pack, tmpdir, portal_mock):
     assert 0 == stbt_rig.main([
         'stbt_rig.py', '--node-id=mynode', '--portal-url=%s' % portal_mock.url,
         '--portal-auth-file=token', 'run', 'tests/test.py::test_my_tests'])
+
+
+def test_run_tests_pytest(test_pack, tmpdir, portal_mock):
+    import platform
+    if platform.system() == "Windows":
+        # On appveyor the check_call fails because it can't find py.test.  I'm
+        # sure this is fixable but I don't have a Windows environment to play
+        # with it right now.
+        pytest.skip()
```

Comment:
Does `python py.test ...` work?

Reply:
`py.test` isn't a python file so it shouldn't, but maybe `python -m py.test` will...